LICENCE.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+<!-- See http://creativecommons.org/licenses/by-sa/4.0/legalcode -->
+<!-- Human-readable summary: http://creativecommons.org/licenses/by-sa/4.0/ --> 
+<head>
+  <title>Creative Commons — Attribution-ShareAlike 4.0 International — CC BY-SA 4.0</title>
```

Comment:
This license has a couple of issues. First of all, we don't use copyleft licenses for the GPII, so if we need to be using a CC license, it should not be the ShareAlike version. Secondly, what does the license apply to? Just the HTML content of this Website? We should probably link to this on CC's website rather than keeping a copy in our repository.